libs/application/templates/financial-aid/src/forms/ApplicationForm/personalInterestSection/inARelationshipSubsection.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  buildCheckboxField,
+  buildDescriptionField,
+  buildMultiField,
+  buildSubSection,
+  buildTextField,
+  getValueViaPath,
+} from '@island.is/application/core'
+import { Routes } from '../../../lib/constants'
+import * as m from '../../../lib/messages'
+import { NationalRegistrySpouse } from '@island.is/api/schema'
+
+export const inARelationshipSubsection = buildSubSection({
```

Comment:
We are using 'inRelationship', and sometimes using 'inARelationship', maybe better stick with inRelationship?